composer.json

```diff
@@ -78,5 +78,59 @@
             "ergebnis/composer-normalize": true
         },
         "sort-packages": true
+    },
+    "scripts": {
+        "cs:check": "@php php-cs-fixer fix --dry-run --diff",
+        "cs:fix": "@php php-cs-fixer fix",
+        "dev-tools:check": [
+            "./dev-tools/check_file_permissions.sh",
+            "./dev-tools/check_trailing_spaces.sh"
+        ],
+        "dev-tools:docs": "@php dev-tools/doc.php",
+        "dev-tools:install": "@composer -d dev-tools install",
+        "docs": "@dev-tools:docs",
+        "mess-detector": "@php dev-tools/vendor/bin/phpmd . ansi phpmd.xml",
+        "phpstan": "@php -d memory_limit=256M dev-tools/vendor/bin/phpstan analyse",
+        "phpstan:baseline": "@php -d memory_limit=256M dev-tools/vendor/bin/phpstan analyse --generate-baseline=./dev-tools/phpstan/baseline.php",
+        "qa": "@quality-assurance",
+        "quality-assurance": [
+            "Composer\\Config::disableProcessTimeout",
+            "@sa",
+            "@test"
+        ],
+        "sa": "@static-analysis",
+        "static-analysis": [
```

Comment:
why not `mess-detector`, chmod, etc?

Reply:
Because it does not work on PHP 8.2 (in CI it's run on 8.1), so it would fail whole `composer sa` script or would require running it always on <=8.1.

<img width="1653" alt="image" src="https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/assets/600668/fa497db1-e9b2-4276-9165-4b137d01dae8">

I don't know what "chmod" you're talking about.